samples/java-spring-eventsourced-customer-registry/src/main/java/customer/api/CustomerEntity.java

```diff
@@ -19,17 +19,23 @@
 public class CustomerEntity extends EventSourcedEntity<Customer, CustomerEvent> {
   private static final Logger logger = LoggerFactory.getLogger(CustomerEntity.class);
 
+  record Confirm(String msg){
+    public static Confirm done = new Confirm("done");
+  }
+
   @GetMapping
   public Effect<Customer> getCustomer() {
     return effects().reply(currentState());
   }
 
   @PostMapping("/create")
-  public Effect<String> create(@RequestBody Customer customer) {
+  public Effect<Confirm> create(@RequestBody Customer customer) {
```

Comment:
I change this to return `Confirm` instead of raw `String`. The reason is that when calling through the HTTP endpoint, we get: `\"OK\"`. 

Then, when calling from the customer-registry-action, we get: `\"\"OK\"\"`. Not cool.